src/msg.c

```diff
@@ -1081,7 +1081,7 @@ static bool handle_device_added(struct session *session, const char *device_id,
 	msg_credential_create(&msg, device_pending->uuid, token);
```

Comment:
in the commit 5fa54fdd8b99688ff1eae81c422860086102cbea
Fix payload_len when sending a schema and a register resp

Reply:
Done